WebApplication/ClientApp/src/components/forgePdfView.js

```diff
@@ -80,32 +80,35 @@ export class ForgePdfView extends Component {
     componentDidUpdate(prevProps) {
         if (this.viewer && Autodesk && (this.props.drawingPdf !== prevProps.drawingPdf)) {
 
+            const findModelsForUrn = function(viewer, urn, onlyFirstModel) {
+                const allModels = viewer.getAllModels();
+                const models = [];
+                for (const model of allModels) {
+                    const modelUrn = model.getData().urn;
+                    if (modelUrn != urn)
+                        continue;
+
+                    models.push(model);
+
+                    if (onlyFirstModel === true)
```

Comment:
Sorry, I know that my comment sounds too peaky, but
https://github.com/airbnb/javascript#comparison--shortcuts
👼